一直报错panic: runtime error: invalid memory address or nil pointer dereference

## 问题反馈

`nohup: ignoring input
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x774eef]

goroutine 1 [running]:
github.com/charmbracelet/log.(*Logger).log(0x8?, 0x961fd60?, {0xe62f20?, 0xc0057f8b88?}, {0x0?, 0xe85cc0?, 0xc00961fd01?})
	/go/pkg/mod/github.com/charmbracelet/log@v0.2.1/logger.go:52 +0x4f
github.com/charmbracelet/log.(*Logger).Fatal(0xc009718f00?, {0xe62f20?, 0xc0057f8b88?}, {0x0?, 0x203000?, 0xc000003860?})
	/go/pkg/mod/github.com/charmbracelet/log@v0.2.1/logger.go:318 +0x3c
github.com/eryajf/chatgpt-dingtalk/pkg/logger.Fatal(...)
	/github/workspace/pkg/logger/logger.go:41
github.com/eryajf/chatgpt-dingtalk/config.LoadConfig()
	/github/workspace/config/config.go:264 +0x276
github.com/eryajf/chatgpt-dingtalk/public.InitSvc()
	/github/workspace/public/public.go:20 +0x1d
main.init.0()
	/github/workspace/main.go:23 +0x19`

实在是不知道那里问题了，直接用的config.yml
就配置了：
run_mode: "stream"
api_key:“********”
base_url:“********”
app_secrets:[***********]
credentials:
下面参数

大佬们，帮忙看看。。。。

--- a/config/config.go
+++ b/config/config.go
@@ -257,11 +257,10 @@ func LoadConfig() *Configuration {
 	if config.ChatType == "" {
 		config.ChatType = "0"
 	}
-	if config.ApiKey == "" {
-		logger.Fatal("config err: api key required")
-	}
-	if config.ServiceURL == "" {
-		logger.Fatal("config err: service url required")
+	if !config.AzureOn {
+		if config.ApiKey == "" {
+			panic("config err: api key required")
+		}
 	}
 	if config.MaxQuestionLen == 0 {
 		config.MaxQuestionLen = 4096

